### PR TITLE
Add documentation for routing Host introduced in 0.18.2

### PR DESCRIPTION
**Pull request by saurori**
The `Host` function was introduced in [v0.18.2](https://github.com/gobuffalo/buffalo/releases/tag/v0.18.2) from [this PR](https://github.com/gobuffalo/buffalo/pull/2185).

I've added documentation for English and attempted a translation to French via Google Translate.

### CODE REVIEW

**Review by dmuriel — APPROVED** [review]
Thanks @saurori for the documentation, I like it.

File: templates/fr/docs/routing/_hosts.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+## Hôtes
+
+<%= sinceVersion("0.18.2") %>
+
+Buffalo prennent également en charge le regroupement des points de terminaison par hôte. L'hôte crée un nouveau groupe qui correspond au domaine transmis. Ceci est utile pour créer des groupes de points de terminaison pour différents domaines ou sous-domaines.
```
Comment (stanislas-m) [inline templates/fr/docs/routing/_hosts.md:5]: Host creates a new group that matches the domain passed. This is useful for creating groups of end-points for different domains or subdomains.
```suggestion
Buffalo supporte également le regroupement d'APIs par domaine. La méthode `Host` crée un nouveau groupe pour le nom de domaine passé en paramètre, ce qui peut être une méthode pratique pour regrouper des APIs appartenant au même domaine (voire gérer plusieurs sous-domaines sur une même application).
```

**Review by stanislas-m — CHANGES_REQUESTED** [review]

File: templates/en/docs/routing/_hosts.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+## Hosts
+
+<%= sinceVersion("0.18.2") %>
+
+Buffalo apps also support grouping of end-points by host. Host creates a new group that matches the domain passed. This is useful for creating groups of end-points for different domains or subdomains.
```
Comment (stanislas-m) [inline templates/en/docs/routing/_hosts.md:5]:
```suggestion
Buffalo apps also support grouping of end-points by host. `Host` creates a new group that matches the domain passed. This is useful for creating groups of end-points for different domains or subdomains.
```

**Review by stanislas-m — CHANGES_REQUESTED** [review]

File: templates/fr/docs/routing/_hosts.md (hunk 1 above)
Comment (saurori) [reply templates/fr/docs/routing/_hosts.md]: Updated in https://github.com/gobuffalo/docs/pull/605/commits/d4b35fe13f94431f3aa51537735d4e1fd5d59348

File: templates/en/docs/routing/_hosts.md (hunk 2 above)
Comment (saurori) [reply templates/en/docs/routing/_hosts.md]: Updated in https://github.com/gobuffalo/docs/pull/605/commits/d4b35fe13f94431f3aa51537735d4e1fd5d59348

File: templates/fr/docs/routing/_hosts.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+domainApp.GET("/", func (c buffalo.Context) error {
+  return c.Render(http.StatusOK, r.String("example.com Homepage"))
+})
+
+app.GET("/", func (c buffalo.Context) error {
+  return c.Render(http.StatusOK, r.String("Main App Homepage"))
+})
+```
+
+Les variables mappées aux paramètres sont également prises en charge:
```
[the remark this answers is not in the document]
Comment (saurori) [reply templates/fr/docs/routing/_hosts.md]: Updated in https://github.com/gobuffalo/docs/pull/605/commits/d4b35fe13f94431f3aa51537735d4e1fd5d59348

**Review by stanislas-m — APPROVED** [review]

**Comment by stanislas-m** [comment]
Thanks @saurori!

**Comment by saurori** [comment]
@stanislas-m just a heads up, the deployment failed: https://github.com/gobuffalo/docs/deployments

**Pull request merged**